meta-mel/conf/distro/include/mel.conf

```diff
@@ -234,6 +234,8 @@ PACKAGECONFIG_REMOVE_pn-mesa-demos = "glx"
 # We don't use dracut to build initramfs
 PACKAGECONFIG_REMOVE_pn-plymouth = "initrd"
 
+# Do not fetch the anti-virus db at compile-time
+INSTALL_CLAMAV_CVD = ""
```

Comment:
Would it make more sense to set this using
`INSTALL_CLAMAV_CVD_mel ?= ""`

Reply:
It's defined in the recipe with ?=, and _mel is redundant when it's already being defiend in mel.conf, unless it's necessary. I do like the idea of using ?= to ease overrides from local.conf, though. I'll adjust it.

Reply:
Updated.